tensorflow/compiler/mlir/tosa/transforms/legalize_common.cc

```diff
@@ -2577,6 +2566,197 @@ llvm::Optional<Value> convertReduceMeanOp(
   return val;
 }
 
+// Lowers ResizeBilinear and ResizeNearestNeighbor to TOSA resize.
+llvm::Optional<Value> convertResizeOp(PatternRewriter& rewriter, Operation* op,
+                                      RankedTensorType output_type,
+                                      Value input_value, StringRef mode,
+                                      bool align_corners,
+                                      bool half_pixel_centers) {
+  RankedTensorType input_type =
+      input_value.getType().dyn_cast<RankedTensorType>();
+  if (!input_type) return llvm::None;
+
+  auto input_shape = input_type.getShape();
+  auto output_shape = output_type.getShape();
+
+  size_t input_height = input_shape[1];
```

Comment:
I recommend making these typed `int64_t`, which is the actual range of the shape components (size_t is not 64 bits on 32 bit platforms). Recommend using a signed type because you are just feeding into signed arithmetic below.

Reply:
Thanks. Will update in next round.